test/IntegrationTests/Helpers/TestHttpServer.AspNetCore.cs

```diff
@@ -56,10 +59,22 @@ public void Dispose()
         _listener.Dispose();
     }
 
+#pragma warning disable SA1204
```

Comment:
can we avoid this?

Reply:
fixed in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/4107/commits/e52867251269ffa2c7048f070d8e9ae01352c2e8